linkis-engineconn-plugins/hbase/hbase-core/src/main/scala/org/apache/linkis/manager/engineplugin/hbase/executor/HBaseEngineConnExecutor.scala

```diff
@@ -0,0 +1,202 @@
+    engineExecutorContext.appendStdout(result.getResult)
+    SuccessExecuteResponse()
+  }
+
+  private def getHBaseRuntimeParams(
+      engineExecutorContext: EngineExecutionContext
+  ): util.Map[String, String] = {
+    val execCodeUser = getExecCodeUser(engineExecutorContext)
+    var executorProperties: util.Map[String, Object] =
+      engineExecutorContext.getProperties
+    if (executorProperties == null) {
+      executorProperties = new util.HashMap[String, Object]()
+    }
+    logger.info(s"executorProperties is: $executorProperties")
+    val globalConfig: util.Map[String, String] = new util.HashMap[String, String]()
+    Utils.tryAndWarn(HBaseEngineConfig.getCacheMap(engineExecutorContext.getLabels))
+    globalConfig.put(HBaseEngineConnConstant.KERBEROS_PROXY_USER, execCodeUser)
+    if (!executorProperties.isEmpty) {
+      val _executorProperties: Map[String, Object] = executorProperties.asScala.toMap
+      _executorProperties.foreach(d => {
+        val k: String = d._1
+        val v: Object = d._2
+        if (k.startsWith(HBaseEngineConnConstant.LINKIS_PREFIX)) {
+          globalConfig.put(k, String.valueOf(v))
+        }
+      })
+    }
+    globalConfig
+  }
+
+  private def getExecCodeUser(engineExecutionContext: EngineExecutionContext): String = {
+    val userCreatorLabel = engineExecutionContext.getLabels
+      .find(_.isInstanceOf[UserCreatorLabel])
+      .get
+      .asInstanceOf[UserCreatorLabel]
+    userCreatorLabel.getUser
+  }
+
+  override def executeCompletely(
+      engineExecutorContext: EngineExecutionContext,
+      code: String,
+      completedLine: String
+  ): ExecuteResponse = null
+
+  override def progress(taskID: String): Float = 0
+
+  override def getProgressInfo(taskID: String): Array[JobProgressInfo] =
+    Array.empty[JobProgressInfo]
+
+  override def supportCallBackLogs(): Boolean = false
+
+  override def getExecutorLabels(): util.List[Label[_]] = executorLabels
+
+  override def setExecutorLabels(labels: util.List[Label[_]]): Unit = {
+    if (!CollectionUtils.isEmpty(labels)) {
+      executorLabels.clear()
+      executorLabels.addAll(labels)
+    }
+  }
+
+  override def requestExpectedResource(expectedResource: NodeResource): NodeResource = null
+
+  override def getCurrentNodeResource(): NodeResource = {
+    NodeResourceUtils.appendMemoryUnitIfMissing(
+      EngineConnObject.getEngineCreationContext.getOptions
+    )
+
+    val resource = new CommonNodeResource
+    val usedResource = new LoadResource(OverloadUtils.getProcessMaxMemory, 1)
+    resource.setUsedResource(usedResource)
+    resource
+  }
+
+  override def getConcurrentLimit: Int = HBaseConfiguration.HBASE_CONCURRENT_LIMIT.getValue
+
+  override def killAll(): Unit = {
+    logger.info("Killing all query task.")
+
+    logger.info("All query task has killed successfully.")
+  }
+
+  override def killTask(taskId: String): Unit = {
+    logger.info(s"Killing hbase query task $taskId")
+    super.killTask(taskId)
```

Comment:
need to support kill task

Reply:
Supports kill task, which will destroy the hbase-shell session. Every time you kill the task and rerun the task, the session will be restarted.

Reply:
> Supports kill task, which will destroy the hbase-shell session. Every time you kill the task and rerun the task, the session will be restarted.

If the task can only be killed by ending the session, I think it is also necessary.